sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -1465,3 +1465,23 @@ def get_avro_schema_from_table_schema(schema):
   dict_table_schema = get_dict_table_schema(schema)
   return bigquery_avro_tools.get_record_schema_from_dict_table_schema(
       "root", dict_table_schema)
+
+
+class BigQueryJobTypes:
+  EXPORT = 'EXPORT'
+  COPY = 'COPY'
+  LOAD = 'LOAD'
+  QUERY = 'QUERY'
+
+
+_BQ_JOB_NAME_TEMPLATE = "beam_bq_job_{job_type}_{job_id}_{step_id}_{random}"
+
+
+def generate_bq_job_name(job_name, step_id, job_type, random=None):
```

Comment:
Can we standardize and define this format in bigquery.py (probably with a backwards compatibility warning) so that users can rely on it ?

Reply:
Done.